### PR TITLE
Add build workfow

### PR DESCRIPTION
**Pull request by claudiamurialdo**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        
+    - name: Configure Azure Artifacts feed
+      if: github.repository_owner == 'GeneXusLabs' && steps.buildVariables.outputs.SHOULD_DEPLOY == 'true'
+      run: |
+        $IsPrerelease = [System.Convert]::ToBoolean($Env:IsPrerelease)
+
+    - name: Push packages
+      if: github.repository_owner == 'GeneXusLabs' && steps.buildVariables.outputs.SHOULD_DEPLOY == 'true'
+      env:
+        GPRFeedURL: https://nuget.pkg.github.com/genexuslabs/index.json
```
Comment (jechague) [inline .github/workflows/build.yml:68]:
```suggestion
```

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        $pushedToNuget  = 0
+
+        Get-ChildItem ".\dotnet\*.nupkg" -Recurse | ForEach-Object {
+          $PushToNugetArgs  = @("nuget", "push", $_.FullName, "--source", $Env:NuGetFeedURL, "--api-key", "${{ secrets.NUGET_ORG_TOKEN }}")
+
+          if ([string]::IsNullOrEmpty("${{ github.event.inputs.skip-duplicates }}") ) {
+            $skipDuplicates = $true
+          } else {
+            $skipDuplicates = [System.Convert]::ToBoolean("${{ github.event.inputs.skip-duplicates }}")
+          }
```
Comment (jechague) [inline .github/workflows/build.yml:83]:
```suggestion
            $skipDuplicates = $true
```

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+        $codesign_pfx = "code_sign_cert.pfx"
+        $bytes = [Convert]::FromBase64String($Env:PFX_BASE64)
+        [IO.File]::WriteAllBytes($codesign_pfx, $bytes)
+
+        Get-ChildItem ".\nuget\*.nupkg" -Recurse | ForEach-Object {
+          dotnet nuget sign $_.FullName --certificate-path $codesign_pfx --certificate-password $Env:PFX_PASS --timestamper $Env:TIMESTAMPER_URL
+        }
+        
+    - name: Configure Azure Artifacts feed
+      if: github.repository_owner == 'GeneXusLabs' && steps.buildVariables.outputs.SHOULD_DEPLOY == 'true'
```
Comment (jechague) [inline .github/workflows/build.yml:61]: The `buildVariables` step is missing.

If the plan is to push a new version of the package each time a commit is made to the branch, this condition can be simplify to just check if the workflow is executing in a pull request context or not.

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+      - name: Dispatch build result
+        uses: ./tmp/.github/actions/dispatch-build-result
+        with:
+          component-name: ${{ github.event.inputs.repository }}
+          branch-ref: ${{ env.GIT_REF }}
+          new-version: ${{ needs.build.outputs.NUGET_VERSION }}
+          committer: ${{ needs.build.outputs.LAST_COMMITTER }}
+          commit-message: ${{ needs.build.outputs.COMMIT_MESSAGE }}
+          token: ${{ secrets.SECURE_TOKEN }}
```
Comment (jechague) [inline .github/workflows/build.yml:129]:
```suggestion
```

If I understood correctly, the package published by this repository will be used as a dependency in the DotNet Classes repo

**Review by jechague — CHANGES_REQUESTED** [review]
Left some comments about the new workflow.

Moreover, remember that the package prefix must be either _"GeneXus"_ or _"genexus"_ since those are the prefixes reserved to us by NuGet.

File: .github/workflows/build.yml (hunk 3 above)
Comment (claudiamurialdo) [reply .github/workflows/build.yml:61]: Ok. Condition changed.

**Comment by claudiamurialdo** [comment]
> Left some comments about the new workflow.
> 
> Moreover, remember that the package prefix must be either _"GeneXus"_ or _"genexus"_ since those are the prefixes reserved to us by NuGet.

Done.

**Review by jechague — APPROVED** [review]
LGTM

**Pull request merged**